src/application.ts

```diff
@@ -107,7 +111,8 @@ export default class Application implements Owner {
     if (!mainTemplate) { throw new Error("Could not find main template."); }
 
     let mainLayout = templateFactory(mainTemplate).create(this.env);
```

Comment:
we should kill the idea of a `mainTemplate` and just hard code this into glimmer-application:

```hbs
{{#each roots key="id" as |root|}}
  {{#-in-element root.element nextSibling=root.nextSibling}}
    {{component root.component}}
  {{/-in-element}}
{{/each}}
```